qos-monitor/src/main/java/eu/arrowhead/core/qos/dto/event/EventDTOConverter.java

```diff
@@ -0,0 +1,195 @@
+package eu.arrowhead.core.qos.dto.event;
+
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+import java.util.List;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.util.Assert;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+
+import eu.arrowhead.common.Utilities;
+import eu.arrowhead.common.dto.shared.EventDTO;
+import eu.arrowhead.common.dto.shared.IcmpPingResponseDTO;
+import eu.arrowhead.common.dto.shared.QosMonitorEventType;
+import eu.arrowhead.common.exception.InvalidParameterException;
+import eu.arrowhead.core.qos.QosMonitorConstants;
+import eu.arrowhead.core.qos.dto.event.monitoringevents.FinishedMonitoringMeasurementEventDTO;
+import eu.arrowhead.core.qos.dto.event.monitoringevents.InterruptedMonitoringMeasurementEventDTO;
+import eu.arrowhead.core.qos.dto.event.monitoringevents.ReceivedMonitoringRequestEventDTO;
+import eu.arrowhead.core.qos.dto.event.monitoringevents.StartedMonitoringMeasurementEventDTO;
+
+public class EventDTOConverter {
+
+	//=================================================================================================
+	// members
+
+	private static final Logger logger = LogManager.getLogger(EventDTOConverter.class);
+
+	private static final ObjectMapper mapper = new ObjectMapper();
+
+	static {
+		mapper.configure(SerializationFeature.INDENT_OUTPUT, true);
+	}
+
+	//=================================================================================================
+	// methods
+
+	//-------------------------------------------------------------------------------------------------
+	public static InterruptedMonitoringMeasurementEventDTO convertToInteruptedMonitoringMeasurementEvent(final EventDTO event) {
```

Comment:
typo in method name: interupted => interrupted

Reply:
fixed